packages/frontend-core/src/constants.js

```diff
@@ -63,3 +63,25 @@ export const TableNames = {
  *   - Coerce types for search endpoint
  */
 export const ApiVersion = "1"
+
+/**
+ * Maximum minimum range for SQL number values
+ */
+export const SQL_NUMBER_TYPE_RANGE_MAP = {
```

Comment:
NAB: could we use camel case here rather than block caps to stick with the convention?